cppsdk/source/playfab/PlayFabEvent.cpp

```diff
@@ -0,0 +1,70 @@
+#include <stdafx.h>
+
+#ifndef DISABLE_PLAYFABENTITY_API
+
+#include <playfab/PlayFabEvent.h>
+
+namespace PlayFab
+{
+    void PlayFabEvent::SetName(const std::string& eventName)
```

Comment:
Is there a task for unit testing, maybe validate input?

Reply:
Yes, there are task for end-to-end testing and for important unit tests

---
In reply to: [227990937](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227990937) [](ancestors = 227990937)